core/blockchain.go

```diff
@@ -239,18 +239,20 @@ func (bc *BlockChain) loadLastState() error {
 	bc.currentFastBlock = bc.currentBlock
 	if head := GetHeadFastBlockHash(bc.db); head != (common.Hash{}) {
 		if block := bc.GetBlockByHash(head); block != nil {
-			bc.currentFastBlock = block
+			bc.currentFastBlock.Store(block)
 		}
 	}
 
 	// Issue a status log for the user
+	currentFastBlock := bc.currentFastBlock.Load().(*types.Block)
```

Comment:
Could `bc.currentFastBlock.Load().(*types.Block)` be replaced with `bc.CurrentFastBlock()`?

Seems you're doing it below, wondering why not here.